include/mgclient.h

```diff
@@ -15,7 +15,17 @@
 #ifndef MGCLIENT_MGCLIENT_H
 #define MGCLIENT_MGCLIENT_H
 
+#ifdef EMSCRIPTEN
+#define MGCLIENT_EXPORT
+#else
 #include "mgclient-export.h"
+#endif
+
+#ifdef __EMSCRIPTEN__
```

Comment:
What is the difference between `EMSCRIPTEN` and `__EMSCRIPTEN__`?

Reply:
The first is wrong, I quote:

>The preprocessor define __EMSCRIPTEN__ is always defined when compiling programs with Emscripten.

Reply:
Lol github hides `_` :roll_eyes: Plz refer to https://emscripten.org/docs/compiling/Building-Projects.html